src/containers/twitter-embed/twitter.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./twitter.css";
+import {TwitterTimelineEmbed} from "react-twitter-embed";
+import { twitterDetails } from "../../portfolio";
+
+var widthScreen=window.screen.width;
+export default function Twitter() {
+	if (twitterDetails.userName){
+		return (
+            <div class="tw-main-div" id="twitter">       
+			<div className="centerContent">
+			<TwitterTimelineEmbed
+				sourceType="profile"
+				screenName ={ twitterDetails.userName }
+				options={{height:400,width: {widthScreen}}}
+				placeholder="Loading twitter Component Please Wait."
```

Comment:
Since I couldn't view the component because of privacy protection, we should replace the placeholder with something like,
"My latest Twitter activity (Can't load? Check privacy protection settings)"

Reply:
Done!